tests/next_tests/integration_tests/multi_feature_tests/iterator_tests/test_vertical_advection.py

```diff
@@ -42,22 +42,17 @@ def tridiag_backward2(x_kp1, cp, dp):
 
 
 @fundef
-def solve_tridiag(a, b, c, d):
-    cpdp = lift(scan(tridiag_forward, True, make_tuple(0.0, 0.0)))(a, b, c, d)
-    return scan(tridiag_backward, False, 0.0)(cpdp)
-
-
-def tuple_get_it(i, x):
-    def stencil(x):
-        return tuple_get(i, deref(x))
-
-    return lift(stencil)(x)
+def solve_tridiag(domain, a, b, c, d):
+    cpdp = as_fieldop(scan(tridiag_forward, True, make_tuple(0.0, 0.0)), domain)(a, b, c, d)
```

Comment:
is this because the lift inliner is not active?

Reply:
The only scan pattern supported by gtfn is `as_fieldop(scan(...), ...)` so I switched to that pattern in the ITIR tests.